homeassistant/components/otbr/config_flow.py

```diff
@@ -55,6 +55,6 @@ async def async_step_hassio(self, discovery_info: HassioServiceInfo) -> FlowResu
         url = f"http://{config['host']}:{config['port']}"
         await self.async_set_unique_id(DOMAIN)
         return self.async_create_entry(
-            title="Open Thread Border Router",
+            title=discovery_info.name,
```

Comment:
If we're on a Yellow or SkyConnect, we should make that the title.

Reply:
That would maybe be nice, but how do we know if the OTBR is using the home assistant yellow or sky connect radio?

To know that, we should add some kind of source when the addon install is initiated by the yellow or sky connect integration, and then check that here.

Otherwise, we can check if we're on a yellow or sky connect, and the addon is configured to use that.

Reply:
The latter sounds like a good idea 👍